### ISSUE TITLE
The old zh_cn.lang is not compatible in the newer Minecraft version

### ISSUE THREAD

**Issue by Demonese** [jaquadro/StorageDrawers#988]
I create 3 pull requests, they can solve this problem.
I also update zh_cn translations. All entries are translated.

* 1.16 #987 
* 1.17 #986 
* 1.18 #985 

I also noticed some pull requests that have not been merged for a long time:

* 1.16 #965 (some translations are missing)
* 1.16 #915 (some translations are missing)
* 1.16 #858 (some translations are missing)
* 1.15 #837 
* 1.15 #834 (zh_tw)

Thanks!

**Comment by jaquadro** [comment]
Yeah most activity on this project was on pause for the last 1-2 year.  Then if multiple translations appear for the same language it creates a bit of paralysis because I can't personally evaluate them.

In this case, it looks like there's 5 (!!) for zh_cn.
